src/viadot/sources/sql_server.py

```diff
@@ -0,0 +1,110 @@
+import struct
+from datetime import datetime, timedelta, timezone
+from typing import List
+from pydantic import BaseModel, SecretStr
+
+from viadot.sources.base import SQL
+from viadot.config import get_source_credentials
+
+
+class SQLServerCredentials(BaseModel):
+    user: str
+    password: str | SecretStr = None
+    server: str
+    driver: str = "ODBC Driver 17 for SQL Server"
```

Comment:
Is this source work in container ? I remember that this driver should be installed in container to be able to connect to sqlserver